www/src/pages/foundations/typography.tsx

```diff
@@ -42,356 +101,233 @@ const measuredTypeProps = {
+              className,
+              text,
+            }) => ({ className, text }))}
+            columns={[
+              {
+                Header: 'Style',
+                Cell: ({ row }: DataTableTypes) => (
+                  <p className={row.original.className} style={{ fontSize: '16px' }}>
+                    {row.original.text}
+                  </p>
+                ),
+              },
+              {
+                Header: 'CSS Class',
+                Cell: ({ row }: DataTableTypes) => (
+                  <code style={{ fontSize: '16px' }}>
+                    .{row.original.className}
+                  </code>
+                ),
+              },
+            ]}
+          >
+            <DataTable.Table />
+          </DataTable>
+        </div>
 
-        <table className="w-100 table pgn-doc__status-table">
-          <tbody>
-            <tr>
-              <th colSpan={2}>
-                <h2>Alignment</h2>
-              </th>
-            </tr>
-            <tr>
-              <th>Style</th>
-              <th>CSS Class</th>
-            </tr>
-            <tr>
-              <td>
-                <p className="text-left">left text.</p>
-              </td>
-              <td>
-                <code>.text-left</code>
-              </td>
-            </tr>
-            <tr>
-              <td>
-                <p className="text-right">right text.</p>
-              </td>
-              <td>
-                <code>.text-right</code>
-              </td>
-            </tr>
-            <tr>
-              <td>
-                <p className="text-center">center text.</p>
-              </td>
-              <td>
-                <code>.text-center</code>
-              </td>
-            </tr>
-            <tr>
-              <td>
-                <p className="text-justify">justify text.</p>
-                <p
-                  className="text-justify text-muted small"
-                  style={{ maxWidth: '20rem' }}
-                >
-                  At the edge of forever tendrils of gossamer clouds corpus
-                  callosum culture Vangelis dispassionate extraterrestrial
-                  observer.
-                </p>
-              </td>
-              <td>
-                <code>.text-justify</code>
-              </td>
-            </tr>
-            <tr>
-              <td>
-                <p className="text-wrap">wrap text.</p>
-              </td>
-              <td>
-                <code>.text-wrap</code>
-              </td>
-            </tr>
-            <tr>
-              <td>
-                <p className="text-nowrap">nowrap text.</p>
-              </td>
-              <td>
-                <code>.text-nowrap</code>
-              </td>
-            </tr>
-          </tbody>
-        </table>
+        <h2 className="mb-4">Alignment</h2>
+        <div className="mb-4">
+          <DataTable
+            itemCount={2}
+            data={alignmentClassesAndDescriptions.map(({
```

Comment:
no need to do `.map` here, `data={alignmentClassesAndDescriptions}` should work

Reply:
Thanks, simplified